### DISCUSSION TITLE
Sharing a website (e.g. for online games)

### DISCUSSION THREAD

**Discussion by VogtAI** [edumeet/edumeet#829] [Ideas]
It would be nice if one could share a link to any website which could be integrated as an iframe in the same style as screen sharing. This way one could share a link to a multiplayer game (simple dice games or card games) one could play with friends. Additionally it would be nice if optionally everyone's nickname could be concatenated to the url, so you wouldn't have to reenter your name in the game.

**Comment by misi** [comment]
Thanks! If you can please send Pull Request based on develop branch. I would happy to merge it.
